src/component/photo-modal/PhotoModal.jsx

```diff
@@ -0,0 +1,139 @@
+import React, {Component} from "react";
+import Api from "../../api/index";
+
+import "./_photo-modal.scss";
+
+const exifLabels = {
+    aperture: "Aperture",
+    exposure_time: "Exposure Time",
+    focal_length: "Focal Length",
+    iso: "ISO",
+    make: "Make",
+    model: "Model"
+};
+
+export default class PhotoModal extends Component {
+
+    state = {
+        loading: true,
+        exif: {},
+        location: {},
+        rawURL: null,
+    };
+
+    async componentDidMount() {
+        const api = new Api();
+
+        const getPhotoFullDetail = await api.photo(this.props.id);
+        const { exif = {}, location = null, urls } = getPhotoFullDetail.data;
+        let getPhotoLocation = null;
+
+        if (location != null) {
+            getPhotoLocation = await api.getStaticMap({
+                ...location
+            });
+        }
+
+        this.setState({
+            loading: false,
+            exif,
+            rawURL: urls.raw,
+            location: location != null ? Object.assign(location, {
+                image: getPhotoLocation
+            }) : null,
+        })
+
+    }
+
+
+    render() {
+        const { 
+            alt_description, 
+            user 
+        } = this.props;
+        const { 
+            username, 
+            name 
+        } = user;
+        const { 
+            exif, 
+            location, 
+            rawURL, 
+            loading 
+        } = this.state;
+        const { 
+            title, 
+            image = null 
+        } = location || {};
```

Comment:
You've already assigned a `{}` default value for `location`. The following version should work the exact same.
```suggestion
        } = location;
```

Reply:
`hizini alamamak`